src/components/eip3643/User.tsx

```diff
@@ -34,6 +36,23 @@ export default function User() {
     });
   }, [deployedTokens, accountEvm, setTokens]);
 
+  const sortTokensList = () => {
```

Comment:
I'd suggest to add a comment or more telling name for the func about how exactly are we sorting the things